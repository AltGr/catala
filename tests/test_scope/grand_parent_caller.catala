@ Article @

/*
new scope A:
  param x type int

new scope B:
  param a scope A
  param y1 type int
  param y2 type int

new scope C:
  param b scope B
  param z1 type int
  param z2 type int


scope A:
  def x := 0

scope B:
  def a.x := y1
  def y1 := 1
  def y2 := a.x

scope C:
  def b.y1 := z1
  def z1 := 2
  def z2 := b.y2
*/