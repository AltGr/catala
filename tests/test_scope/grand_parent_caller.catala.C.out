[RESULT] z1 -> 2
[RESULT] z2 -> 2
